java-connectors/kafka-connect-common/src/main/java/io/lenses/streamreactor/common/util/JarManifest.java

```diff
@@ -69,11 +73,13 @@ public JarManifest(JarFile jarFile) {
     Manifest manifest;
     try {
       Optional<JarFile> jarFileOptional = of(jarFile);
-      manifest = jarFileOptional.get().getManifest();
+      if (jarFileOptional.isPresent()) {
```

Comment:
this check is redundant, as the Option will be present as we are using `of()` instead of `ofNullable()`

Reply:
yup, I changed that